lib/actions/merge.js

```diff
@@ -37,6 +47,8 @@ class Merge extends Action {
       throw new UnSupportedSettingError(`Unknown Merge method, supported options are ${MERGE_METHOD_OPTIONS.join(', ')}`)
     }
     const mergeMethod = settings.merge_method ? settings.merge_method : 'merge'
+    const commitTitle = settings.commit_title ? settings.commit_title : ''
```

Comment:
if you default it like this, wouldn't the default commit title be an empty string? same with commit message?
have you tested out the behavior when the template is not present in settings? I think it's important for backward compatibility

Reply:
I added a test case to confirm this works as expected when the user doesn't set title or message, and it does. But what this also means is that the user can't set an empty message in YAML. So I went ahead and also updated the code to support empty strings. Thanks for catching this!